packages/terra-application-utility/src/terra-dev-site/test/application-utility/DefaultApplicationMenuUtility.test.scss

```diff
@@ -0,0 +1,8 @@
+@import './ApplicationUtilityTestCommon';
+
+:local {
+  .content-wrapper {
+    @extend %common-application-content-wrapper;
```

Comment:
Lets directly add the styles that we are using here instead of using the @extends syntax. I think this add extra complexity to these examples that aren't needed.

Reply:
Done